cmake/Modules/FindComputeCpp.cmake

```diff
@@ -275,12 +293,14 @@ function(add_sycl_to_target targetName binaryDir sourceFiles)
   )
   # Add custom target to run compute++ and generate the integration header
   foreach(sourceFile ${sourceFiles})
+    message (STATUS "binaryDir = ${binaryDir}")
```

Comment:
Was this a debugging print?

Reply:
My bad. Remaining debug message. Will remove.